src/ecr_check_repos_for_terraform/ecr_check_repos_for_terraform/config.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+import argparse
+import logging
+
+import ecr_check_repos_for_terraform.context
+from lib.logging import setup_logging
+
+class AppConfig:
+  def __init__(self, cmdline_args, env):
+    ''' set app-wide configs and initialize the app '''
+    loglevel = (
+      cmdline_args.loglevel
+    )
+
+    self._init_app(loglevel)
+
+  def _init_app(self, loglevel):
+    ''' initialize the app '''
+    # initialize logging
+    setup_logging(eval('logging.' + loglevel))
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+  )
+  parser.add_argument(
+    '--loglevel',
+    choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+    default='INFO',
+    help='log level'
+  )
+  return parser.parse_args()
+
+def parse_env(env):
+  ''' parse and validate env vars '''
+  pass
+
+# import this from main script
+# object will be instantiated only once
+config = AppConfig(parse_args(), parse_env(os.environ))
```

Comment:
mostly boiler plate. since the script doesn't require any env vars. loglevel is the only cmdline param.